src/test/java/org/mockito/configuration/MockitoConfiguration.java

```diff
@@ -0,0 +1,12 @@
+package org.mockito.configuration;
+
+/**
+ * This is to clean runners between tests.
+ */
+public class MockitoConfiguration extends DefaultMockitoConfiguration {
```

Comment:
This is to instruct the VM to unload classes that are no longer used. It may release some memory in big test suites. Here doesn't have much difference.